src/app/components/home/home.component.scss

```diff
@@ -13,11 +13,23 @@
   max-width:25%;
   padding:0;
   float:left;//parent div width = child div's width
-
   x-tab{
     height:auto;
     padding:20px;
     justify-content: left;
-    border-bottom:0.5px solid white;
+    border-bottom:0.5px solid teal;
```

Comment:
Revert this back to white for now. I'll look into changing the highlighting for the NavBar at a later time, just add it under the "features we want" section in Trello.